config/locales/client.en.yml

```diff
@@ -50,7 +50,7 @@ en:
 
   start_proposal_card:
     title: 'Start a proposal'
-    helptext: Ready to make a decision? Proposals let you see how everyone feels about a particular course of action.
+    helptext: Ready to make a decision?
```

Comment:
I really like the "See how people feel about a course of action" part.. I wonder if there is room for that bit, @rdbartlett